src/index.js

```diff
@@ -18,21 +20,26 @@ const isRNStorybook =
 export default function testStorySnapshots (options = {}) {
   if (isStorybook) {
     storybook = require.requireActual('@kadira/storybook')
+    const loadBabelConfig = require('@kadira/storybook/dist/server/babel_config').default
     const configDirPath = path.resolve(options.configPath || '.storybook')
     configPath = path.join(configDirPath, 'config.js')
+
+    const content = babel.transformFileSync(configPath, babelConfig).code
+    const contextOpts = {
+      filename: configPath,
+      dirname: configDirPath
+    }
+    const babelConfig = loadBabelConfig(configDirPath)
```

Comment:
Note that `babelConfig` declared at line 32 is unused, and the reference to `babelConfig` at line 27 will be evaluated to undefined.